src/applications/personalization/profile/components/personal-information/ContactInformationField.jsx

```diff
@@ -261,22 +267,6 @@ class ContactInformationField extends React.Component {
       </div>,
     );
 
-    if (isEmpty) {
```

Comment:
This component no longer has special handling for the "no-data" state. Instead the `ContactInformationView` component (used above on line 247) has been expanded to handle the "no-data" state.